### ISSUE TITLE
Empty directory thirdparty/gtest/googletest

### ISSUE THREAD

**Issue by jwuttke** [gpospelov/qt-mvvm#48]
Similar issue #3 which was closed but not resolved.

CMake fails to build:

```
$ cmake ..
-- The CXX compiler identification is GNU 9.2.1
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
--   Qt5::Widgets /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5.12.5
--   Includes: /usr/include/x86_64-linux-gnu/qt5/;/usr/include/x86_64-linux-gnu/qt5/QtWidgets;/usr/include/x86_64-linux-gnu/qt5/QtGui;/usr/include/x86_64-linux-gnu/qt5/QtCore;/usr/lib/x86_64-linux-gnu/qt5//mkspecs/linux-g++;/usr/include
CMake Error at thirdparty/gtest/CMakeLists.txt:10 (add_subdirectory):
  The source directory

    /G/ext/qt-mvvm/thirdparty/gtest/googletest

  does not contain a CMakeLists.txt file.


-- Configuring incomplete, errors occurred!
```

Actually, in my fresh clone the directory thirdparty/gtest/googletest is empty.

In the Github source browser, I do see a link to the upstream googletest repository. This link, however, seems to be ignored by `git clone` on my Debian/testing system.

**Comment by jwuttke** [comment]
When fixing this, make sure that `make install` does not install gtest and gmock.

**Comment by gpospelov** [comment]
Resolved in #57
